README.md

```diff
@@ -58,37 +58,48 @@ Create and deploy an app-operator using the SDK CLI:
 ```sh
 # Create an app-operator project that defines the App CR.
 $ mkdir -p $GOPATH/src/github.com/example-inc/
+# Create a new app-operator project
 $ cd $GOPATH/src/github.com/example-inc/
-$ operator-sdk new app-operator --api-version=app.example.com/v1alpha1 --kind=App
+$ operator-sdk new app-operator
 $ cd app-operator
 
+# Add a new API for the custom resource AppService
+$ operator-sdk add api --api-version=app.example.com/v1alpha1 --kind=AppService
```

Comment:
I am not sure why there are now 3 commands when you could use only one before. Was this in high demand so people just generate what they need. If so why not use flags instead of adding a subcommand. Just curious about the reasons behind this.

Reply:
This was wanted for ansible operator. When an ansible operator wants to watch a new resource it only needs a CRD. Making an option to short circuit the API command and make it more clear to our users how to add the resources they will need was important to us. 

This was already implemented in the generate command and was moved to `add` as it is a better fit. I think that if we want to remove this option can we open an issue and/or set up a discussion?

Reply:
Oh, yes was just a question. Thanks!